ibis-server/tools/generate_openapi.py

```
@@ -0,0 +1,51 @@
+# This script generates the OpenAPI specification for the application and saves it to a file.
```

Ruff (0.8.2)

File `ibis-server/tools/generate_openapi.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)